tests/regression/lp-2061981/test-snapd-sh/meta/snap.yaml

```diff
@@ -0,0 +1,13 @@
+name: test-snapd-sh
+summary: A no-strings-attached, no-fuss shell for writing tests
+version: 1.0
+
+plugs:
+    personal-files:
+        read: [$HOME/myhome]
```

Comment:
Is `$HOME` here `/home` or `/home/myhome/test2`? AFAICT the test expects to find `/home/myhome/test2/testfile`, so in neither case would granting `$HOME/myhome` seem to be correct, unless it grants recursive read access and `$HOME` is `/home`. But in that case, what happens if the `personal-files` plug grants `$HOME/myhome/test2` instead, does it still fail?

Reply:
$HOME is user home directory I understand

Reply:
I udpated the snap to allow read from $HOME and I get the error 

grep error: pattern not found, got:
cannot update snap namespace: cannot expand mount entry (none $HOME/.missing none x-snapd.kind=ensure-dir,x-snapd.must-exist-dir=$HOME 0 0): cannot use invalid home directory "/home/myhome/test2": permission denied